resources/monitoring/charts/alert-rules/templates/kyma-rules.yaml

```diff
@@ -0,0 +1,43 @@
+{{ define "kyma-rules.yaml.tpl" }}
+groups:
+- name: pod-not-running-rule
+  rules:
+  - alert: SystemPodNotRunning
+    expr: (kube_pod_container_status_running { namespace=~"kyma-.*|kube-.*|istio-.*", pod!~"(test.*)|((dummy|sample)-.*)|(.*(docs|backup|test)-.*)|(.*-(tests|dummy))" } == 0)
+    for: 60s
+    labels:
+      severity: critical
+    annotations:
+      description: "{{`{{ $labels.namespace }}`}}/{{`{{ $labels.pod }}`}} is not running"
+      summary: "{{`{{ $labels.pod }}`}} is not running"
+- name: cpu-90-percent-usage-rule
+  rules:
+  - alert: CPU90PercentUsage
+    expr:   sum (rate(container_cpu_usage_seconds_total{namespace="kyma-system"}[2m])) by (pod_name) * on(pod_name) group_left(app) label_replace(max(kube_pod_labels{label_alertcpu=~"yes*"}) by (pod), "pod_name", "$1", "pod", "(.*)") * 100 > 90
+    for: 10m
+    labels:
+      severity: critical
+    annotations:
+      description:  "Pod {{`{{$labels.pod}}`}} is having {{`{{$value}}`}} % usage"
+      summary: "Pod {{`{{$labels.pod}}`}} is having {{`{{$value}}`}} % usage"
```

Comment:
![screen shot 2019-02-06 at 16 48 04](https://user-images.githubusercontent.com/5954057/52353574-212df400-2a2f-11e9-884d-afff3b84be64.png)
Looks like pod name and namespace are not appearing in `CPU90Percentage` and `MEM90PercentUsage` alerts.